cfme/common/provider_views.py

```diff
@@ -276,13 +277,18 @@ class ContainersProvidersView(ProvidersView):
     """
      represents Main view displaying all Containers providers
     """
+    SUMMARY_TEXT = 'Containers Providers'
     table = Table(locator="//div[@id='list_grid']//table")
 
     @property
+    def paginator(self):
+        return self.entities.paginator
```

Comment:
Q: what is it for ?

Reply:
There is only one paginator so I thought maybe have a shortcut, but we can remove that if you want.